docs/user_guide/data_types_and_io/dataclass.md

```diff
@@ -11,8 +11,31 @@ When you've multiple values that you want to send across Flyte entities, you can
 Flytekit uses the [Mashumaro library](https://github.com/Fatal1ty/mashumaro)
 to serialize and deserialize dataclasses.
 
+With the 1.14 release, `flytekit` adopted `MessagePack` as the 
+serialization format for dataclasses, overcoming  a major limitation of serialization into a JSON string within a Protobuf `struct` datatype, like the previous versions do:
+
+to store `int` types, Protobuf's `struct` converts them to `float`, forcing users to write boilerplate code to work around this issue.
```

Comment:
no need to insert a new line

Reply:
I think insert a new line is more readable